src/transformers/models/bart/modeling_bart.py

```diff
@@ -171,7 +171,10 @@ def forward(
         # if key_value_states are provided this layer is used as a cross-attention layer
         # for the decoder
         is_cross_attention = key_value_states is not None
-        bsz, tgt_len, embed_dim = hidden_states.size()
+
+        # Use the class's parameter as the hidden_state's last dimension.
+        # This dimension cannot be used in case of enabling tensor-parallelism.
```

Comment:
I don't think this comment is useful when reading the new code. It creates more confusion than help, only the next one is really important.

Reply:
I agree, I can remove this.